doc/examples/uuid.rst

```diff
@@ -281,13 +322,43 @@ when encoding UUIDs from all drivers. UUIDs written by a driver with this
 representation configured will be handled correctly by every other provided
 it is also configured with the ``STANDARD`` representation.
 
-``STANDARD`` encodes native :class:`uuid.UUID`s to
+``STANDARD`` encodes native :class:`uuid.UUID` objects to
 :class:`~bson.binary.Binary` subtype 4 objects.
 
+.. attention:: In PyMongo 3.x, applications can end up inadvertently changing
+   the :class:`~bson.binary.Binary` subtype of a UUID field when round-tripping
+   documents with legacy-encoded UUIDs. This happens because in PyMongo 3.x,
+   the ``STANDARD`` representation decodes all UUIDs (subtypes 3 and 4) as
```

Comment:
The opposite is also true, I think? If you retrieve a document with subtype 4 fields but have any of the legacy representations configured?

Reply:
Ah, that's correct. I will break this out into a separate section dedicated to problems that can be caused by using the incorrect representation.

Reply:
Done.